src/libs/actions/ReportActions.ts

```diff
@@ -56,6 +59,19 @@ function clearReportActionErrors(reportID: string, reportAction: ReportAction, k
     });
 }
 
+const reportActionsByReport: OnyxCollection<ReportActions> = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
```

Comment:
None that I know of 😅 

Thanks for catching that. It probably should be using `waitForCollectionCallback` since that will have a slight performance improvement.

@EzraEllette Could you do a follow-up PR to modify that?